src/components/ImportFile/ImportFileContent/ImportFileContent.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import {
+  ImportFileAccept,
+  ImportFileActive,
+  ImportFilePassive,
+  ImportFileReject
+} from './index';
+
+const files = [];
+
+test('ImportFileAccept is working properly', () => {
```

Comment:
Please move each of these into their own files so that tests can be targeted.  This file should test the ImportFileContent component and the results of the switch statement in the render method.

Reply:
Most of the project's component are creating 1 test file, just trying to follow the guidelines here. Is 1 per file new since using `enzyme`?

Reply:
Sorry about this, I might have jumped the gun with this comment.  It looks like testing best practices are being revisited.  Any suggestions/recommendations will end up being tracked here.https://github.com/patternfly/patternfly-react/issues/287. For now you can ignore the testing comments that go against the current repo's setup/design.